server/routes/appointments/appointmentSummaryComponent.ts

```diff
@@ -0,0 +1,72 @@
+import { SummaryListItem } from '../../utils/summaryList'
+import { ActionPlanAppointment } from '../../models/actionPlan'
+import Appointment from '../../models/appointment'
+import AppointmentDecorator from '../../decorators/appointmentDecorator'
+import PresenterUtils from '../../utils/presenterUtils'
+import AuthUserDetails from '../../models/hmppsAuth/authUserDetails'
+
+export default class AppointmentSummaryComponent {
```

Comment:
I think we can drop the `Component` from the name here as we don't tend to use `component` much in our codebase - I initially thought `AppointmentSummaryPresenter` might be better, but equally `AppointmentSummary` could work on its own? What do you think?

Reply:
(we do have AddressFromComponent -  I don't think it matters much, but having a consistent way to differentiate between something that is 1-2-1 linked to a page and just a general component for use would be nice)

I've renamed to AppointmentSummary